### ISSUE TITLE
Sales Data Download Page Accessibility Fix: Royalty Equation as data table

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#3249]
As found via Trusted Tester, the royalty equation in step one of the process diagram is coded as a data table within a data table. WCAG does not allow data tables to be used for anything other than tables with headers and related cells. 

https://revenuedata.doi.gov/downloads/federal-sales/

Here's a little bit of info on data vs layout tables and making sure a layout table is properly linearized:

https://www.csun.edu/universal-design-center/web-accessibility-criteria-tables

https://www.w3.org/TR/WCAG20-TECHS/F49.html

<img width="726" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNzVoQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--44120e71e3cdb238881ae4e93ebf0d12a256adbc/image.png" alt="image.png" />

**Comment by eelzi-ONRR** [comment]
@AlexandracmONRR will provide an image and alt-text for the equation

**Comment by AlexandracmONRR** [comment]
Also, provided on slack, but also will provide here on GitHub.

Alt Text: 

Royalty due equals open parenthesis sales volume multiplied by unit value multiplied by royalty rate close parenthesis subtract allowances. The Royalty Value Prior to allowances (RVPA) includes sales volume multiplied by unit value multiplied by royalty rate. Royalty Value Less Allowances (RVLA) includes open parenthesis sales volume multiplied by unit value multiplied by royalty rate close parenthesis subtract allowances.

![ProcessDiagram-Step1-Equation](https://github.com/DOI-ONRR/nrrd/assets/122935097/7d6d2880-e0f0-4579-9c8e-178860d594b8)
